include/battle_interface.h

```diff
@@ -79,14 +76,10 @@ enum
 #define TAG_STELLAR_INDICATOR_TILE      0xD792
 #define TAG_TERA_TRIGGER_TILE           0xD793
 
-#define TAG_MEGA_TRIGGER_PAL            0xD777
+#define TAG_GIMMICK_TRIGGER_PAL         0xD777
 #define TAG_MEGA_INDICATOR_PAL          0xD778
 #define TAG_MISC_INDICATOR_PAL          0xD779 // Alpha, Omega, and Dynamax indicators use the same palette as each of them only uses 4 different colors.
-#define TAG_ZMOVE_TRIGGER_PAL           0xD77B
-#define TAG_BURST_TRIGGER_PAL           0xD77C
-#define TAG_DYNAMAX_TRIGGER_PAL         0xD77D
 #define TAG_TERA_INDICATOR_PAL          0xD77E
```

Comment:
```suggestion
#define TAG_TERA_INDICATOR_PAL          0xD77B
```
Same as above (maybe even `0xD77A`? That seems to be unused for palettes?)